sections/featured-products-preview.liquid

```diff
@@ -0,0 +1,150 @@
+{{ 'hot-featured-products.css' | asset_url | stylesheet_tag }}
+
+{% assign id = 100 | rand: 2000 %}
+
+{% style %}
+  .section-featured-products-{{id}} {
+    padding-top: {{ section.settings.padding_top }}px;
+    padding-bottom: {{ section.settings.padding_bottom }}px;
+  }
+  .section-featured-products-{{id}} .yc-hot-products {
+    justify-content: {{ section.settings.section_title_position }};
+  }
+{% endstyle %}
+
+<div class='yc-featured-products section-featured-products-{{id}} container'>
+    <div class="yc-hot-products">
+      <img 
+          loading='lazy'
+          src='{%- if section.settings.image.src -%} {{ section.settings.image.src }} {%- endif -%}'
```

Comment:
If the default image value is null, it will display a crashing picture, make sure to either default it or execute the check before the tag